src/nytid/cli/hr.nw

```diff
@@ -221,6 +242,9 @@ def get_canvas_users(username_regex, course_regex):
 All the filtering we want to do has already been implemented in [[canvaslms]].
 So we can simply call the right library functions.
 <<filter out [[courses]] in Canvas from [[course_regex]]>>=
+canvas_session = get_canvas_session()
+if canvas_session is None:
+  return []
```

Comment:
If the canvas session is not available, we should raise an exception. The user must set up canvas and ladok for this to work.